### PR TITLE
92/change password

### PR DESCRIPTION
**Pull request by rawstorage**
added change password and incremented minor version

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -1,104 +1 @@
-# flake8: noqa
```
Comment (helenwalsh) [inline docs/source/conf.py]: Not sure what this is about.  Was it your intention to change this file

File: PyU4V/system.py [hunk 2]
```diff
@@ -924,3 +925,28 @@ def get_ip_interface(self, director_id, port_id, interface_id,
...
+        """Function to allow users to change password for local user accounts.
+
+        Requires minimum version of Unisphere 9.2.1.x and API account must
+        have security admin role
+
+        Change local user password.
+        :param username: username for local account -- str
+        :param current_password: existing password for local account -- str
+        :param new_password: new password for local account -- str
+
```
Comment (helenwalsh) [inline PyU4V/system.py:940]: Gray area.  Should it return a dict from the modify_resource?

File: docs/source/conf.py (hunk 1 above)
Comment (helenwalsh) [reply docs/source/conf.py]: With .gitattribute they should check in as linux.

Comment (helenwalsh) [reply docs/source/conf.py]: Its in a strange format

File: PyU4V/system.py (hunk 2 above)
Comment (helenwalsh) [reply PyU4V/system.py:940]: :returns: -- dict

**Review by helenwalsh — CHANGES_REQUESTED** [review]
nit picking again.  take a look at the conf.py.  Not sure what is going on here.  Its in a weird format.  I guess you only wanted to change the version in it.

File: PyU4V/system.py (hunk 2 above)
Comment (rawstorage) [reply PyU4V/system.py:940]: There is actually no return on this, it will return None if successful

File: ChangeLog [hunk 3]
```diff
@@ -1,5 +1,9 @@
 PyU4V Change Log
 ================
+Version 9.2.1.2
+- support for changing local users passwords via REST, requires Unipshere 9.2.1
+  changchange_local_user_password
```
Comment (helenwalsh) [inline ChangeLog]: Apologies, missed this earlier.  typo changchange_local_user_password

**Review by helenwalsh — CHANGES_REQUESTED** [review]
typo in the change log.  Once that is fixed I will approve

**Review by helenwalsh — APPROVED** [review]
If all tests and pep8 are clean I approve

**Comment by rawstorage** [comment]
merged and closed

**Pull request merged**